### ISSUE TITLE
crash telegram desktop 1.7.10

### ISSUE THREAD

**Issue by Black-mass** [telegramdesktop/tdesktop#6214]
### Actual behaviour
There is a bug in the desktop version of the Telegram  that causes the crash and closure of the program. This bug occurs because when you open the 
@tgdata

 channel link and the second post from the end of the animation and tgs files in the program occurs

### Configuration
** All Operating system:**

**Version of Telegram Desktop 1.7.10:**

**Comment by erotavlas85** [comment]
I have a similar crash during the opening of @tgbeta
This is the log file
cat /home/user/.TelegramDesktop/log.txt 
```
[2019.07.05 09:15:50] Launched version: 1007010, install beta: [TRUE], alpha: 0, debug mode: [TRUE], test dc: [FALSE]
[2019.07.05 09:15:50] Executable dir: /home/user/Telegram/, name: Telegram
[2019.07.05 09:15:50] Initial working dir: /home/user/
[2019.07.05 09:15:50] Working dir: /home/user/.TelegramDesktop/
[2019.07.05 09:15:50] Command line: /home/user/Telegram/Telegram
[2019.07.05 09:15:50] Executable path before check: /home/user/Telegram/Telegram
[2019.07.05 09:15:50] Logs started
[2019.07.05 09:15:50] Fontconfig version: 2.11.94.
[2019.07.05 09:15:50] Connecting local socket to /tmp/38ae22053d2c10675fda247087363414-{87A94AB0-E370-4cde-98D3-ACC110C5967D}...
[2019.07.05 09:15:50] Socket connect error 0, starting server and app...
[2019.07.05 09:15:50] Moved logging from '/home/user/.TelegramDesktop/log_start0.txt' to '/home/user/.TelegramDesktop/log.txt'!
[2019.07.05 09:15:50] Opened '/home/user/.TelegramDesktop/tdata/working' for reading, the previous Telegram Desktop launch was not finished properly :( Crash log size: 3040
[2019.07.05 09:15:50] Update Info: MTP is unavailable.
[2019.07.05 09:15:51] App Error: _basePath is empty in writeSettings()
[2019.07.05 09:15:51] Primary screen DPI: 96.1263
[2019.07.05 09:15:51] Could not load 'ayatana-appindicator3' with version 1 :(
[2019.07.05 09:15:51] Error: failed to load 'gdk_x11_drawable_get_xdisplay' function!
[2019.07.05 09:15:51] Getting DE, XDG_CURRENT_DESKTOP: 'MATE'
[2019.07.05 09:15:51] Getting DE, DESKTOP_SESSION: 'mate'
[2019.07.05 09:15:51] Getting DE, KDE_SESSION_VERSION: ''
[2019.07.05 09:15:51] DE: Gnome
[2019.07.05 09:15:51] Tray Icon: Try Qt = [TRUE], Prefer appindicator = [FALSE]
[2019.07.05 09:15:51] App Info: reading settings...
[2019.07.05 09:15:51] App Info: reading encrypted settings...
[2019.07.05 09:15:51] Lang Info: Loaded cached, keys: 2397
[2019.07.05 09:15:51] Lang Error: Unexpected tag (key 'lng_call_box_status_group:count')
[2019.07.05 09:15:51] Lang Error: Unknown tag (key 'lng_manage_discussion_group_warning:visible')
[2019.07.05 09:15:51] OpenAL Logging Level: (not set)
[2019.07.05 09:15:51] Audio Playback Devices: OpenAL Soft
[2019.07.05 09:15:51] Audio Playback Default Device: OpenAL Soft
[2019.07.05 09:15:51] Audio Capture Devices: Monitor of Built-in Audio Analogue Stereo
[2019.07.05 09:15:51] Audio Capture Default Device: Monitor of Built-in Audio Analogue Stereo
[2019.07.05 09:15:51] App Info: reading map...
[2019.07.05 09:15:51] App Info: reading encrypted map...
[2019.07.05 09:15:51] App Info: reading encrypted user settings...
[2019.07.05 09:15:51] App Info: encrypted user settings read.
[2019.07.05 09:15:51] App Info: reading encrypted mtp data...
[2019.07.05 09:15:51] MTP Info: read keys, current: 5, to destroy: 0
[2019.07.05 09:15:51] Map read time: 23
[2019.07.05 09:15:51] TCP Error: network - Network unreachable
[2019.07.05 09:15:51] HTTP Error: network error 99 - Network unreachable
[2019.07.05 09:15:51] Notifications Server: Notification Daemon
[2019.07.05 09:15:51] LibNotify capabilities: actions, body, body-hyperlinks, body-markup, icon-static, sound
[2019.07.05 09:15:51] Export Info: Destroy top bar by controller removal.
[2019.07.05 09:15:51] Using Qt tray icon.
[2019.07.05 09:15:51] Tray Icon: Using Qt tray icon, available: [TRUE]
[2019.07.05 09:15:51] Not using Unity Launcher count.
[2019.07.05 09:15:56] Exception: unknown cons 0x5931
[2019.07.05 09:16:03] TCP Error: network - Network unreachable
[2019.07.05 09:16:03] HTTP Error: network error 99 - Network unreachable
[2019.07.05 09:16:03] TCP Error: network - Network unreachable
[2019.07.05 09:16:03] TCP Error: network - Network unreachable
[2019.07.05 09:16:03] HTTP Error: network error 99 - Network unreachable
[2019.07.05 09:16:03] TCP Error: network - Network unreachable
[2019.07.05 09:16:03] HTTP Error: network error 99 - Network unreachable
[2019.07.05 09:16:03] HTTP Error: network error 99 - Network unreachable
[2019.07.05 09:16:05] Exception: unknown cons 0x1
[2019.07.05 09:16:05] Exception: unknown cons 0x9
**[2019.07.05 09:20:06] Assertion Failed! "_type != 0" scheme.cpp:19299**
```

**Comment by Aokromes** [comment]
fixed on 1.7.11 if you cannot update because telegram desktop, archive @tgbeta @tgdata from mobile phone and start telegram desktop.
